--- a/package/Config.in	2013-09-25 17:42:50.173431598 +0200
+++ b/package/Config.in	2013-09-25 17:43:58.557429564 +0200
@@ -553,8 +553,11 @@
 menu "Networking"
 source "package/czmq/Config.in"
 source "package/cppzmq/Config.in"
+source "package/dtndht/Config.in"
 source "package/filemq/Config.in"
 source "package/glib-networking/Config.in"
+source "package/ibrcommon/Config.in"
+source "package/ibrdtn/Config.in"
 source "package/libcgi/Config.in"
 source "package/libcgicc/Config.in"
 source "package/libcurl/Config.in"
@@ -702,6 +705,7 @@
 source "package/snowball-init/Config.in"
 source "package/sound-theme-borealis/Config.in"
 source "package/sound-theme-freedesktop/Config.in"
+source "package/wifisd-legacy/Config.in"
 endmenu
 
 menu "Networking applications"
@@ -741,6 +745,8 @@
 source "package/hiawatha/Config.in"
 source "package/hostapd/Config.in"
 source "package/httping/Config.in"
+source "package/ibrdtnd/Config.in"
+source "package/ibrdtn-tools/Config.in"
 if BR2_PACKAGE_BUSYBOX_SHOW_OTHERS
 source "package/ifplugd/Config.in"
 endif
